shell/platform/android/test/io/flutter/embedding/engine/renderer/FlutterRendererTest.java

```diff
@@ -621,6 +621,25 @@ public void ImageReaderSurfaceProducerImageReadersAndImagesCount() {
     assertEquals(0, texture.numImages());
   }
 
+  // A 0x0 ImageReader is a runtime error.
+  @Test
+  public void ImageReaderSurfaceProducerClampsWidthAndHeightTo1() {
+    FlutterRenderer flutterRenderer = new FlutterRenderer(fakeFlutterJNI);
+    FlutterRenderer.ImageReaderSurfaceProducer texture =
+        flutterRenderer.new ImageReaderSurfaceProducer(0);
+
+    // Default values.
+    assertEquals(texture.getWidth(), 1);
+    assertEquals(texture.getHeight(), 1);
+
+    // Try setting width and height to 0.
+    texture.setSize(0, 0);
```

Comment:
I would call getSurface after setSize(0,0) and make sure that it doesn't raise an exception.

Reply:
Done!